tests/WebAppCallsWebApiCallsGraph/Client/appsettings.json

```diff
@@ -29,6 +29,17 @@
         "BaseUrl": "https://localhost:44351"
 
     },
+    "SayHello": {
+        /*
+      TodoListScope is the scope of the Web API you want to call. This can be: "api://a4c2469b-cf84-4145-8f5f-cb7bacf814bc/access_as_user",
+      - a scope for a V2 application (for instance api://b3682cc7-8b30-4bd2-aaba-080c6bf0fd31/access_as_user)
+      - a scope corresponding to a V1 application (for instance <GUID>/user_impersonation, where  <GUID> is the
+        clientId of a V1 application, created in the https://portal.azure.com portal.
+    */
+        "Scopes": "api://a4c2469b-cf84-4145-8f5f-cb7bacf814bc/access_as_user", // "api://712ae8d7-548a-4306-95b6-ee9117ee86f0/access_as_user",
```

Comment:
Do we want to keep the comment? what was it?

Reply:
which comment? i see them all there